controller/src/main/java/io/pravega/controller/server/rpc/grpc/v1/ControllerServiceImpl.java

```diff
@@ -144,6 +146,67 @@ public void getCurrentSegmentsKeyValueTable(KeyValueTableInfo request, StreamObs
                 responseObserver);
     }
 
+    @Override
+    public void listKeyValueTablesInScope(Controller.KVTablesInScopeRequest request, StreamObserver<Controller.KVTablesInScopeResponse> responseObserver) {
+        String scopeName = request.getScope().getScope();
+        RequestTag requestTag = requestTracker.initializeAndTrackRequestTag(requestIdGenerator.get(),
+                "listKeyValueTables", scopeName);
+        log.info(requestTag.getRequestId(), "listKeyValueTables called for scope {}.", scopeName);
+
+        final AuthContext ctx;
+        if (this.grpcAuthHelper.isAuthEnabled()) {
+            ctx = AuthContext.current();
+        } else {
+            ctx = null;
+        }
+
+        authenticateExecuteAndProcessResults(
+                () -> {
+                    String result = this.grpcAuthHelper.checkAuthorization(
+                            AuthResourceRepresentation.ofScope(scopeName),
+                            AuthHandler.Permissions.READ,
+                            ctx);
+                    log.debug("Result of authorization for [{}] and READ permission is: [{}]",
```

Comment:
Do we have a consistent structure for the logging related to KV tables in controller? If not, it may be a good idea, so we can apply reggex expressions with `grep` when looking into the logs.

Reply:
This method is used by both streams and KVTables.